### DISCUSSION TITLE
Sample Configuration

### DISCUSSION THREAD

**Discussion by weiserman** [miggi92/odata-fw#131] [Ideas]
This is a really interesting project!
Are there any samples or examples that one could follow? 
There is no documentation on how to define OData (customizing) properties, just need a little more guidance to set up a sample project, class and generate a basic OData service.

Thanks Warren

**Comment by miggi92** [comment]
Hello Warren, 

thanks for your input. I'm going to create a basic example as soon as i have time for it. Currently I'm working on different projects.

Best regards
Miguel

**Comment by miggi92** [comment]
@weiserman  I added a first implementation example to the documentation. Hope this can help you to understand how the framework works. 
[https://miggi92.github.io/odata-fw/documentation/examples/material.html](https://miggi92.github.io/odata-fw/documentation/examples/material.html )

**Reply by weiserman** [reply]
Thank you very much for the additional information. 

Was able to create a basic business partner service on developer 1909 Edition!
![CleanShot 2024-08-22 at 15 45 18@2x](https://github.com/user-attachments/assets/8bfd65ad-d5a6-487b-863d-247d923db3e0)
